arch/x86/kernel/tsc.c

```diff
@@ -280,6 +280,16 @@ static int __init tsc_setup(char *str)
 
 __setup("tsc=", tsc_setup);
 
+static int tsc_freq_override_khz;
+
+static int __init tsc_freq_setup(char *str)
+{
+	tsc_freq_override_khz = simple_strtol(str, NULL, 0);
+	return 1;
```

Comment:
Hi,
Why always `return 1`?
Couldn't you just replace the return type by `static void`?